README.md

```diff
@@ -123,12 +130,12 @@ We automatically deploy this app with [Heroku](https://www.heroku.com/).
 * [Olofvi](https://github.com/olofvi)
 * [Sajaas](https://github.com/Sajaas)
 
-### License
+## License
 MIT license
 
-### Acknowledgments
-The entire project is comprised of open source code so a huge **THANKS!** to the open source community. 
-Built with help from the coaches and some cool gems. Thanks a lot! Coaches:
+## Acknowledgments
+The entire project is comprised of open source code so a huge **THANKS** to the open source community. 
+Built with help from the coaches and some cool gems.
```

Comment:
Don't we want to have Gustaf as well here?

Reply:
He's still on the list.